### ISSUE TITLE
mongodb-memory-server MongoDB Compass compability

### ISSUE THREAD

**Issue by suchcodemuchwow** [nodkz/mongodb-memory-server#402]
## Versions

- NodeJS: 0.0.0
- mongodb-memory-server-*: 0.0.0
- mongodb: 0.0.0
- mongoose: 0.0.0 
- system: 

package: mongo-memory-server 

## What is your question?

Is this package compatible with compass ? If it's can you guys share some information about what should be valid connection string ?

**Comment by hasezoey** [comment]
why shouldnt it be compatible? this package just spins up an mongod process with the provided options,
if you need the connection string, try `console.log("uri", mongoServer.getUri())` or dosnt this uri work?

otherwise: i personally use Compass (Community Edition), and there you just need to enter the IP or DNS with the correct port, you can get the port with `console.log("port", mongoServer.getPort())`

(replace `mongoServer` with the variable you saved the instance of `MongoMemoryServer` in)

**Comment by synthmusic** [comment]
Things to pay attention to:  make sure you are using the port from the latest launch of your app.

In my case, using nodemon, it's easy to save a file and lose the fact that there's now a new instance on a different port. I ended up on this issue because I couldn't connect, and then realized this.

P.S. I saved myself a lot of getting connect time by setting a specific port in the options. Compass reconnects gracefully without needing to click connect:
https://github.com/nodkz/mongodb-memory-server#available-options-for-mongomemoryserver
